### PR TITLE
add react1 week 3 homework

### PR DESCRIPTION
**Pull request by aladdin91**
this is my react1 week 3 homework

### CODE REVIEW

File: react/react1/week3/todo-app/src/App.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+li {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  border: solid black 2px;
+  width: 80%;
+  margin: 6px;
+}
+ul {
```
Comment (Divyajg) [inline react/react1/week3/todo-app/src/App.css:13]: Styling HTML elements directly like `li` and `ul` is not a good practice. Try using classnames/ids instead.

File: react/react1/week3/todo-app/src/components/AddTodo.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    if (isNaN(deadlineDate.getTime())) {
+      alert("Please input a valid date for the deadline.");
+      return;
+    }
+
+    // Check if deadline is in the future
+    const now = new Date();
+    if (deadlineDate < now) {
+      alert(
+        "We don't have time machine yet, until then please input a valid date :)."
```
Comment (Divyajg) [inline react/react1/week3/todo-app/src/components/AddTodo.jsx:34]: I like this humor ❤

**Review by Divyajg — APPROVED** [review]
Good work. 👍

**Comment by aladdin91** [comment]
thanks a lot for the review 🙏

**Pull request merged**